core/src/main/java/rdx/works/core/domain/resources/Resource.kt

```diff
@@ -155,6 +155,7 @@ sealed class Resource {
     data class NonFungibleResource(
         override val address: ResourceAddress,
         val amount: Long,
+        val displayAmount: Long,
```

Comment:
The `amount` is being used for pagination logic, therefore it shouldn't be altered. However, we needed another field: `displayAmount` - to represent the amount visible to the user in the UI.